src/main/kotlin/no/nav/syfo/sykmelding/api/SykmeldingApi.kt

```diff
@@ -26,6 +28,23 @@ fun Route.registerSykmeldingApiV2() {
         call.respond(sykmeldinger)
     }
 
+    get("/sykmeldinger/{sykmeldingid}/tidligere-arbeidsgivere") {
+        val sykmeldingId = call.parameters["sykmeldingid"]!!
+        val principal: BrukerPrincipal = call.authentication.principal()!!
+        val fnr = principal.fnr
+        if (sykmeldingId == "null") {
+            logger.warn("Mottok kall for å hente sykmelding med id null, sender 404 Not Found")
```

Comment:
"Sender 404 not found" høyres rart ut kanskje returnerer?

Reply:
ja det er mer en bad request

Reply:
ja det er mer en bad request

Reply:
tror også at det ikke går an, fordi den vil aldri treffe denne pathen med null id